luaunit.lua

```diff
@@ -1279,19 +1264,12 @@ then OK or FAILED (failures=1, error=1)
 
 ]]
 
-local TextOutput = { __class__ = 'TextOutput' }
-local TextOutput_MT = { -- class
-    __index = TextOutput
-}
+local TextOutput = { __class__ = 'TextOutput' } -- class
+local TextOutput_MT = { __index = TextOutput } -- metatable
 
     function TextOutput:new()
-        local t = {}
-        t.runner = nil
-        t.result = nil
-        t.errorList ={}
-        t.verbosity = M.VERBOSITY_DEFAULT
-        setmetatable( t, TextOutput_MT )
-        return t
+        return setmetatable(
```

Comment:
Something gets lost along the way : the explicit (useless) initialization was a good way to remind the reader of what fields are actually used within the pseudo-object. With the new syntax, it's not clear that fields runner and result are actually available....

Applies also to above classes.